src/app/ApplicationTemplate.Shared/CoreStartup.cs

```diff
@@ -74,15 +75,23 @@ private async Task ExecuteInitialNavigation(CancellationToken ct, IServiceProvid
 
 			if (currentSettings.IsOnboardingCompleted)
 			{
-				await section.Navigate(ct, () => new DadJokesPageViewModel());
+				var isAuthenticated = await authenticationService.GetAndObserveIsAuthenticated().FirstAsync(ct);
+				if (isAuthenticated)
+				{
+					await services.GetRequiredService<IStackNavigator>().NavigateAndClear(ct, () => new DadJokesPageViewModel());
+				}
+				else
+				{
+					await section.Navigate(ct, () => new LoginPageViewModel(isFirstLogin: false));
+				}
 			}
 			else
 			{
 				await section.Navigate(ct, () => new OnboardingPageViewModel());
 			}
-//-:cnd:noEmit
+			//-:cnd:noEmit
```

Comment:
Should you really add spaces here? Did you test that it still works by creating a new app with the template?